src/index.ts

```diff
@@ -49,6 +49,15 @@ export class Sql {
       const child = rawValues[i++];
       const rawString = rawStrings[i];
 
+      // Check for nested `sql` query after IN keyword.
+      if (rawStrings[i - 1].endsWith("IN (")) {
```

Comment:
What about someone who just wants to use the library for a quick query like `WHERE x IN (1, 2, 3)`?